### PR TITLE
remove debugging output “reading ticks <%s>”

### PR DESCRIPTION
**Pull request by mirabilos**
(if it’s not going to be used/fixed, it’s annoying on the console)

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: libmscore/xmlreader.cpp [hunk 1]
```diff
@@ -185,7 +185,6 @@ Fraction XmlReader::readFraction()
       if (!s.isEmpty()) {
             int i = s.indexOf('/');
             if (i == -1) {
-                  qDebug("reading ticks <%s>", qPrintable(s));
```
Comment (Jojo-Schmitz) [inline libmscore/xmlreader.cpp:188]: It is seen in Debug builds only though, isn't it?

Comment (mirabilos) [reply libmscore/xmlreader.cpp:188]: [“This function does nothing if `QT_NO_DEBUG_OUTPUT` was defined during compilation.”](https://doc.qt.io/Qt-5/qtglobal.html#qDebug)

I see. `QT_NO_DEBUG_OUTPUT` is defined for RELEASE builds. For some reason, Debian builds are neither DEBUG nor RELEASE but "" (empty string).

Feel free to disregard this or merge it, I don’t mind either way.

Comment (Jojo-Schmitz) [reply libmscore/xmlreader.cpp:188]: I guess Debian should do "RELEASE" builds then?

**Comment by mirabilos** [comment]
Joachim Schmitz dixit:

>I guess Debian should do "RELEASE" builds then?

Not in my hands, unfortunately. Debian provides a wrapper
that calls cmake “correctly” and handles probably dozens
of things automatically which I don’t even know of, maybe
even cross compiling, but that wrapper also selects this
build type, and when I asked this seems to be normal.

But, as I said, I can live with that. Feel free to close this PR.

**Comment by Jojo-Schmitz** [comment]
Maybe we should set `QT_NO_DEBUG_OUTPUT` in non-DEBUG builds then?

**Comment by Jojo-Schmitz** [comment]
Does Debian builds disable the asserts?

**Comment by mirabilos** [comment]
Joachim Schmitz dixit:

>Maybe we should set `QT_NO_DEBUG_OUTPUT` in non-DEBUG builds then?

Unsure. The messages themselves can help. If someone doesn’t want
them, they can add it to CPPFLAGS themselves. Better to not force
that.

>Does Debian builds disable the asserts?

Yes.

**Comment by vpereverzev** [comment]
The very annoying one and also it doesn't provide any necessary information, even in debug mode.
Merged

**Pull request merged**